job/test/test_validator.py

```diff
@@ -167,6 +170,7 @@ def test_off_grid_defaults_overrides(self):
         validator.cross_clean()
         self.assertEquals(validator.is_valid, True)
 
-        self.assertAlmostEqual(validator.models["ElectricLoad"].critical_load_pct, 0.95)
+        self.assertAlmostEqual(validator.models["ElectricLoad"].critical_load_pct, 1.0)
+        self.assertAlmostEqual(validator.models["Wind"].operating_reserve_required_pct, 0.1)
```

Comment:
why is the wind operating reserve requirement always 0.1 regardless of what the user sets it to? What's the point of the input then?

Reply:
Good point, we have a few inputs like this where we want to override user inputs or have that input be 0.0. We still want to have them as inputs so users can get the value used by REopt when they GET their responses.

Reply:
@rathod-b the wind `operating_reserve_required_pct` should **not** be overridden if `off_grid_flag=True`. The default should be **0.5** but a user should be able to provide another value. If on-grid, it **should** be forced to be 0.

Reply:
Understood, i was wondering if that was the case for `operating_reserve_required_pct` for PV and ElectricLoad models as well?

Reply:
Addressed in https://github.com/NREL/REopt_API/pull/343/commits/152ac98842eaaf16a7068f0412b34b0893f0886a